concourse/templates/daisy.libsonnet

```diff
@@ -54,7 +55,11 @@
     ] + if self.build_date == '' then
       []
     else
-      ['build_date=' + task.build_date],
+      ['build_date=' + task.build_date
+    ] + if self.sbom_destination == '' then
+      []
+    else
+      ['sbom_destination=' + task.sbom_destination],
```

Comment:
you are checking for `self.sbom_destination` but using `task.sbom_destination` is the intentional?

Reply:
Yeah, I'm copying the format for what was used for "self.build_date" a few lines earlier.